0005-kzen-two-party-ecdsa.adoc

```diff
@@ -0,0 +1,173 @@
+= Kzen Two Party ECDSA Review
+Lloyd Fournier <lloyd@coblox.tech>
+:toc:
+:revdate: 2019-05-10
+
+* Authors(s): {authors}
+* Date: {revdate}
+
+Issue: https://github.com/comit-network/comit-rs/issues/897[#897]
+
+
+== Context
+
+In order to do a _Scriptless Scripts_ atomic swap <<Sanchez18a>> with ECDSA signatures we need to have an implementation of <<Lindell17>>.
+The point of this spike is to review <<Lindell17>> and KZen's rust implementation <<kzen_mp_ecdsa>> and make notes about.
+
+
+== Overview
+
+<<Lindell17>> describes a two-party protocol to (i) generate a joint public key and (ii) jointly create ECDSA signatures under that public key.
+This is usually called signature _aggregation_.
```

Comment:
:upside_down_face: `usually`?
```suggestion
This is called signature _aggregation_.
```

Reply:
Actually the term is wrong. It's called "a signature scheme with key aggregation".

(see https://eprint.iacr.org/2018/068.pdf)